### PR TITLE
ECL-807: Accessility Issues - Added label to form inputs

### PR DESCRIPTION
**Pull request by liandra-giacomine**

### CODE REVIEW

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendReasonView.scala.html [hunk 1]
```diff
@@ -39,12 +39,12 @@
         eclReferenceForStartLink = eclReference
     ) {
 
+        <h1 class="govuk-heading-l">@messages("amendReason.h1")</h1>
```
Comment (TheDanielDoyle) [inline app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendReasonView.scala.html:42]: Errors always have to go above everything. Move the h1 down

File: app/uk/gov/hmrc/economiccrimelevyregistration/viewmodels/govuk/LabelFluency.scala [hunk 2]
```diff
@@ -48,5 +48,11 @@ trait LabelFluency {
 
     def asHiddenPageHeading(size: LabelSize = LabelSize.ExtraLarge): Label =
       asPageHeading(size).withCssClass("govuk-visually-hidden")
+
+    def asHidden(size: LabelSize = LabelSize.ExtraLarge): Label =
+      label
+        .copy(isPageHeading = false)
+        .withCssClass(size.toString)
```
Comment (TheDanielDoyle) [inline app/uk/gov/hmrc/economiccrimelevyregistration/viewmodels/govuk/LabelFluency.scala:55]: If it's a hidden label, why does it need a size class added?

**Review by TheDanielDoyle — CHANGES_REQUESTED** [review]
Please see comments

File: app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendReasonView.scala.html (hunk 1 above)
Comment (paddynski-opencast) [reply app/uk/gov/hmrc/economiccrimelevyregistration/views/AmendReasonView.scala.html]: Done.

**Pull request merged**